instructors.md

```diff
@@ -158,30 +165,35 @@ Solutions to exercises:
 >
 > ~~~ {.python}
 > if 4 > 5:
->     print 'A'
+>     print('A')
 > elif 4 == 5:
->     print 'B'
+>     print('B')
 > elif 4 < 5:
->     print 'C'
-> ~~~
-> ~~~ {.output}
-> C gets printed, because the first two conditions, `4<5` and `4==5` are not true, but `4<5` is true.
+>     print('C')
 > ~~~
+>
+> C gets printed, because the first two conditions, `4 > 5` and `4 == 5` are not true, but `4 < 5` is true.
+>
 
 
 > ## What is truth? {.challenge}
 >
 > After reading and running the code below,
 > explain the rules for which values are considered true and which are considered false.
-> (Note that if the body of a conditional is a single statement, we can write it on the same line as the `if`.)
```

Comment:
This was me.

> Why take this out?

Because from my understanding we want to teach best practices to students and based on that I tough that don't mention this has more positive effects than negative ones.